.pre-commit-config.yaml

```diff
@@ -3,15 +3,6 @@ exclude: |
       .*/test/.*/.*\.approved.json
   )$
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v5.0.0
-    hooks:
-      - id: trailing-whitespace
```

Comment:
I can see we have other formatters in the precommit config.

Are these dropped because having the language specific formatters do their stuff is preferable to these general purpose rules?

Reply:
no, this would be beacause i'm a fool

Reply:
fixed in a fixup!